exec/java-exec/src/main/java/org/apache/drill/exec/store/StoragePluginRegistryImpl.java

```diff
@@ -420,9 +420,13 @@ public void validatedPut(String name, StoragePluginConfig config)
       PluginHandle entry = restoreFromEphemeral(name, config);
       try {
         entry.plugin();
+      } catch (UserException e) {
+        // Provide helpful error messages.
+        throw new PluginException(e.getOriginalMessage(), e);
       } catch (Exception e) {
         throw new PluginException(String.format(
-            "Invalid plugin config for '%s'", name), e);
+            "Invalid plugin config for '%s', "
+          + "Please switch to Logs panel from the UI then check the log.", name), e);
```

Comment:
I think so. there is a link to open the log panel directly?

Reply:
Would a relative link like this work?
```
log/sqlline.log/content
```

Reply:
@cgivre 
`/log/sqlline.log/content` does not contain the error of reconfig plugins. `/log/drillbit.log/content` is the address. However, we need to consider the way to open this page at first, such as open a new tab on brower? or redirect to this page  directly (unfriendly).

Reply:
Is that log visible in the browser?  Maybe just put in the error message the name of the file where the error can be found?

Is there any way to provide more information in the error message itself as to what went wrong?

Reply:
@luocoong
Does the exception message have anything useful in it?  Could we include `e.getMessage()` in the message which is returned?
If you don't think that would be helpful, I'm fine with committing as is.

Reply:
@cgivre Hello. I think this change can alleviate the following problems: 
1. If we create a plugin based on storage plugin (such as `JDBC` `Mongo` `Hbase` `Hive` `Http` and any plugins that extends `AbstractStoragePlugin`), we can get a better error message. because we are actively throw UserException at `org.apache.drill.exec.store.PluginHandle#plugin()`.
2. There is a difference between `getOriginalMessage()` and `getMessage()` in `UserException` class (In fact, getMessage() == ErrorType + Description Text + getOriginalMessage() + Context Message). I prefer the former(more clearly).

Reply:
Hello @luocooong, 
I'm not following you here.  Are you wanting to merge this or are you going to add some additional changes?
Thx!

Reply:
@cgivre Thanks for merging this PR.